src/components/Carousel/Carousel.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from "react";
```

Comment:
```suggestion
import { useState } from "react";
```
리액트 17버전 이후부터는 해당부분을 import 해주지 않아도 돼요 !